src/theme.ts

```diff
@@ -115,13 +115,16 @@ export const space = [
 	128,
 ];
 
-export const fontSizes = [12, 14, 16, 20, 24, 32, 48, 64, 72, 96];
-export const weights = [400, 700];
+// Old font-scale
+// export const fontSizes = [12, 14, 16, 20, 24, 32, 48, 64, 72, 96];
```

Comment:
We can remove this one

Reply:
Thanks @sradevski I'll go through this and fix them

- [ ] 1. The Badges are very small
- [ ] 2. The Card text is quite large compared to the rest of the components
- [ ] 3. Checkbox labels are quite large (compared to Button text, for example)
- [ ] 4. HighlightedName text is quite small
- [ ] 5. Steps text is quite large, especially when compared to the than the Step title

Reply:
@sradevski `HighlightedName`, is already at the size based on the [design](https://app.zeplin.io/project/5cd17644e6012f68816cca3b/screen/5d62783b38345e9b5a66e786). Shall I make it bigger or do we follow the design size?

Reply:
@sradevski I think it's ready to go, let me know how it looks in the dashboard 👍